manifests/export/pem_cert.pp

```diff
@@ -4,30 +4,54 @@
 #
 # == Parameters
 #   [*pfx_cert*]  - PFX certificate/key container
+#   [*der_cert*]  - DER certificate
 #   [*pem_cert*]  - PEM/x509 certificate
 #   [*in_pass*]   - PFX password
 #
-define openssl::export::pem_cert(
-  Stdlib::Absolutepath      $pfx_cert,
-  Stdlib::Absolutepath      $pem_cert = $title,
-  Enum['present', 'absent'] $ensure   = present,
-  Optional[String]          $in_pass  = undef,
+define openssl::export::pem_cert (
+  Optional[Stdlib::Absolutepath]  $pfx_cert = undef,
+  Optional[Stdlib::Absolutepath]  $der_cert = undef,
+  Stdlib::Absolutepath            $pem_cert = $title,
+  Enum['present', 'absent']       $ensure   = present,
+  Optional[String]                $in_pass  = undef,
```

Comment:
Let's reorder the parameters in a more logical way then (`ensure`, then mandatory params, then optional params):

```suggestion
  Enum['present', 'absent']       $ensure   = present,
  Stdlib::Absolutepath            $pem_cert = $title,
  Optional[Stdlib::Absolutepath]  $pfx_cert = undef,
  Optional[Stdlib::Absolutepath]  $der_cert = undef,
  Optional[String]                $in_pass  = undef,
```